client/components/OrderDetails.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+
+class OrderDetails extends Component {
```

Comment:
Does this need to be a class?  Not using class methods or local state.  Can leave if you want.